src/backend/common/helpers/score_breakdown_keys.py

```diff
@@ -334,6 +334,8 @@
             "endGameChargeStationRobot3",
             "endGameParkPoints",
             "extraGamePieceCount",
+            "extraRp1",
+            "extraRp2",
```

Comment:
I'm okay with adding these, but I think the `tba_` prefix will be good to separate them from ones that you should expect from the official breakdown